README.md

```diff
@@ -4,59 +4,75 @@
 [![Coveralls Status](https://img.shields.io/coveralls/FAForever/server/develop.svg)](https://coveralls.io/github/FAForever/server)
 [![semver](https://img.shields.io/badge/license-GPLv3-blue)](license.txt)
 
-This is the source code for the [Forged Alliance Forever](http://www.faforever.com/) server.
+This is the source code for the
+[Forged Alliance Forever](https://www.faforever.com/) lobby server.
 
 # Contributing
 
-To contribute, please fork this repository and make pull requests to the develop branch.
-
-Use the normal git conventions for commit messages, with the following rules:
- - Subject line shorter than 80 characters
- - Proper capitalized sentence as subject line, with no trailing period
- - For non-trivial commits, always include a commit message body, describing the change in detail
- - If there are related issues, reference them in the commit message footer
+Before opening a pull request, please take a moment to look over the
+[contributing guidelines](CONTRIBUTING.md).
 
 ## Setting up for development
 
-First make sure you have an instance of [faf-db](https://github.com/FAForever/db) setup. Then install the dependencies to a virtual environment
-using pipenv by running:
+First, follow the instructions on the [faf-db repo](https://github.com/FAForever/db)
```

Comment:
Sometimes (but rarely) I've had to use a db version different from the latest and the only way we document which version we are assuming is by what is checked out in the `.travis.yml`, I believe. Not sure if that's worth a mention here.

Reply:
I thought about that too, but I think it's too much of an edge case to mention right here. If anything it would go in a section with more details about database migrations, but I'm not even sure a section like that is appropriate for this readme. Maybe it would belong to the readme on the db repo.